archeryutils/handicaps/tests/test_handicaps.py

```diff
@@ -80,65 +74,63 @@
 )
 
 
-@pytest.fixture
-def mocker_hcparams_json(mocker: MockerFixture) -> None:
+class TestHandicapScheme:
     """
-    Override open with a fake HCParams json file.
+    Class to test the handicap_scheme() function creating a HandicapScheme class.
+
+    Methods
+    -------
+    def test_invalid_system()
+        test if invalid handicap system raises error
+
     """
-    mocked_json_file = mocker.mock_open(
-        read_data="""{
-    "AGB_datum": 1.0,
-    "AGB_step": 1.0,
-    "AGB_ang_0": 1.0,
-    "AGB_kd": 1.0,
-
-    "AGBo_datum": 1.0,
-    "AGBo_step": 1.0,
-    "AGBo_ang_0": 1.0,
-    "AGBo_k1": 1.0,
-    "AGBo_k2": 1.0,
-    "AGBo_k3": 1.0,
-    "AGBo_p1": 1.0,
-    "AGBo_arw_d": 3.0,
-
-    "AA_k0": 2.0,
-    "AA_ks": 2.0,
-    "AA_kd": 2.0,
-
-    "AA2_k0": 2.0,
-    "AA2_ks": 2.0,
-    "AA2_f1": 2.0,
-    "AA2_f2": 2.0,
-    "AA2_d0": 2.0,
-
-    "AA_arw_d_out": 3.0,
-    "arrow_diameter_indoors": 3.0,
-    "arrow_diameter_outdoors": 3.0
-    }"""
+
+    def test_invalid_system(self) -> None:
+        """Check that error returned if initialising an invalid system."""
+        with pytest.raises(
+            ValueError,
+            match=(
+                "InvalidSystem is not a recognised handicap system.\n"
+                + "Please select from 'AGB', 'AGBold', 'AA', 'AA2'."
+            ),
+        ):
+            hc.handicap_scheme("InvalidSystem")
+
+    @pytest.mark.parametrize(
+        "scheme",
+        # Check all systems, different distances, negative and large handicaps.
```

Comment:
Leftover comment doesn't apply to this test

Reply:
https://github.com/jatkinson1000/archeryutils/pull/78/commits/fac281c9ceca4541960a00ce009b6d84f50846a6 should resolve this.

Reply:
@TomHall2020 Is this resolved to your satisfaction?